config/initializers/warning.rb

```diff
@@ -0,0 +1,6 @@
+# Avoid this warning since it's unlikely to be fixed upstream.
+# See: https://github.com/podigee/device_detector/issues/91
+if (device_detector_source_path = Gem.loaded_specs["device_detector"]&.full_gem_path)
+  parser_source_file_path = Pathname.new(device_detector_source_path) / "lib/device_detector/parser.rb"
+  Warning.ignore(/regular expression/, parser_source_file_path.to_s)
```

Comment:
In case you also did a double-take here, the warnings generated by `device_detector` that we want to get rid of contain the phrase "regular expression" verbatim, this is not a placeholder.